datadog-monitors-monitor-handler/src/datadog_monitors_monitor/handlers.py

```diff
@@ -137,19 +143,26 @@ def read_handler(
     options = monitor.options if hasattr(monitor, "options") else None
     if options:
         model.Options = MonitorOptions(
-            EnableSamples=options.enable_samples if hasattr(options, "enable_samples") else None,
+            EnableSamples = options.enable_samples if hasattr(options, "enable_samples") else None,
```

Comment:
```suggestion
            EnableSamples=options.enable_samples if hasattr(options, "enable_samples") else None,
```

Reply:
I think  `EnableSamples = options.enable_samples if hasattr(options, "enable_samples") else None` always had space for other options?